src/us-states.js

```diff
@@ -0,0 +1,54 @@
+export var statesData = {"type":"FeatureCollection","features":[
```

Comment:
can we store in in DB or is this the convention as a .js?

Reply:
We can store it in DB ultimately, but right now we need some file to display the effect. Do you want me to add a comment?

Reply:
that's fine, just out of curiousity